packages/oak/lib/core/Col/index.test.js

```diff
@@ -142,8 +142,11 @@ describe('Col', () => {
       }
     ));
 
-    fireEvent.click(container.querySelector('.oak-catalogue .oak-handle'));
-    fireEvent.click(getByText('component 1'));
+    await waitFor(() => {
+      fireEvent.click(container.querySelector('.oak-catalogue .oak-handle'));
+      fireEvent.click(getByText('component 1'));
```

Comment:
why this waitFor ?

Reply:
to avoid act warnings because `checkClipboard` inside `Catalogue`'s `useEffect` is async